framework/doc/content/source/kernels/ConservativeAdvection.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-The `ConservativeAdvection` kernel implements an advection term given for the domain ($\Omega$) define as
+The `ConservativeAdvection` kernel implements an advection term given for the domain ($\Omega$) defined as
 
 \begin{equation}
 \underbrace{\nabla \cdot \vec{v} u}_{\textrm{ConservativeAdvection}} + \sum_{i=1}^n \beta_i = 0 \in \Omega,
```

Comment:
beta term seems irrelevant in this documentation.

Reply:
This beta term was already in the previous documentation.  I had never thought of writing such a term before, but i like it.  With beta you get to specify the whole DE, rather than just the advection part, and it makes signs clearer.